gateway-service/src/main/java/org/zowe/apiml/gateway/security/mapping/ExternalMapper.java

```diff
@@ -73,18 +72,25 @@ MapperResponse callExternalMapper(@NotNull HttpEntity payload) {
             if (httpResponse.getEntity() != null) {
                 response = EntityUtils.toString(httpResponse.getEntity(), StandardCharsets.UTF_8);
             }
-            if (statusCode < HttpStatus.SC_OK || statusCode >= HttpStatus.SC_MULTIPLE_CHOICES) {
-                log.warn("Unexpected response from the external identity mapper. Status: {} body: {}", statusCode, response);
+            if (statusCode == 0) {
```

Comment:
What does it mean? When could it be zero?

Reply:
Based on the condition above, it could be 0 when` httpResponse.getStatusLine()` is null
`org.springframework.http.HttpStatus.valueOf(statusCode)` would then throw exception in this case